kvdb-rocksdb/src/lib.rs

```diff
@@ -303,7 +305,7 @@ impl Database {
 
 		{
 			block_opts.set_block_size(config.compaction.block_size);
-			let cache_size = cmp::max(8, config.memory_budget() / 3);
+			let cache_size = cmp::max(8 * MB, config.memory_budget() / 3);
```

Comment:
I wonder where does this `/ 3` come from, should it be `/ columns`?

Reply:
cc @andresilva (https://github.com/paritytech/parity-ethereum/pull/7348)

Reply:
Nice catch. I think I got that one third from here: https://github.com/facebook/rocksdb/wiki/Setup-Options-and-Basic-Tuning#block-cache-size

Reply:
Actually, I agree with David, that it may be surprising for a user. Although, in practice, most of the time it only makes sense to use the default cache size or to increase it, which won't affect the maximum. I'll push a commit to remove it.